airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml

```diff
@@ -27,14 +27,44 @@ acceptance_tests:
             bypass_reason: "Thumbnail urls changes permanently"
           - name: "ad_creatives"
             bypass_reason: "Thumbnail urls changes permanently"
+          - name: "ads_insights_action_carousel_card"
```

Comment:
how are we testing the new streams if there's no data in our test account?

Reply:
We do have some data for all these streams. Or, more precisely, all ad Insight streams make requests to the same data, but using slicing columns (like in pivot tables).
I've disable them for `basic_read` test, because we use our "live" account for testing.

Reply:
We still test all these streams in `incremental` and `full_refresh` tests